src/MobxPromise.ts

```diff
@@ -97,12 +98,11 @@ export class MobxPromiseImpl<R>
 	private onResult?:(result?:R) => void;
 	private onError?:(error:Error) => void;
 	private defaultResult?:R;
-	private invokeId:number = 0;
-	private _latestInvokeId:number = 0;
+	private _asyncInvokeCount:number = 0;
```

Comment:
better reflects what it is. And we no longer need latestinvokeid and invokeid because we don't have that messiness in status() which was its only purpose AFAIK